src/libraries/Common/src/Interop/Windows/Kernel32/Interop.Constants.cs

```diff
@@ -14,5 +14,7 @@ internal static partial class Kernel32
         internal const int MUTEX_MODIFY_STATE = 0x00000001;
         internal const int SEMAPHORE_MODIFY_STATE = 0x00000002;
         internal const int EVENT_MODIFY_STATE = 0x00000002;
+
+        internal const int LIST_MODULES_ALL = 0x00000003;
```

Comment:
It would be more appropriate to add this constant to `Interop.EnumProcessModulesEx.cs` since it can be only ever used with that API.